### PR TITLE
Add usage examples of <ditafileset>

### PR DESCRIPTION
**Pull request by xephon2**

### CODE REVIEW

File: topics/plugin-coding-conventions.dita [hunk 1]
```diff
@@ -61,6 +61,35 @@
   &lt;ditafileset format="image" />
 &lt;/copy></codeblock>
       </p>
+      <p>The <xmlelement>ditafileset</xmlelement> selector can be used to select different types of
```
Comment (infotexture) [inline topics/plugin-coding-conventions.dita]: Is ”selector” the right term to use for this element?

File: topics/plugin-coding-conventions.dita [hunk 2]
```diff
@@ -61,6 +61,35 @@
...
+        files.</p>
+      <table id="table_dgb_v5p_wcb">
+        <title>Usage examples of &lt;ditafileset></title>
```
Comment (infotexture) [inline topics/plugin-coding-conventions.dita]: This should probably use `<xmlelement>` too.

File: topics/plugin-coding-conventions.dita [hunk 3]
```diff
@@ -61,6 +61,35 @@
...
+            <row>
+              <entry><codeph>&lt;ditafileset format="dita"/></codeph></entry>
+              <entry>Selects all DITA topics in current temporary directory.</entry>
+            </row>
+            <row>
+              <entry><codeph>&lt;ditafileset format="ditamap"/></codeph></entry>
+              <entry>Selects all DITA maps in current temporary directory.</entry>
+            </row>
+            <row>
+              <entry><codeph>&lt;ditafileset format="ditamap"/></codeph></entry>
```
Comment (infotexture) [inline topics/plugin-coding-conventions.dita]: Looks like this should be `format="image"`.

Comment (xephon2) [reply topics/plugin-coding-conventions.dita]: sorry

**Pull request merged**